### PR TITLE
[doc]vertex id 8 bytes

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
### What changes were proposed in this pull request?
as title. There is an error in doc, vertex id is 8 bytes rather than 4. All fixed.

### CODE REVIEW

File: docs/manual-CN/1.overview/3.design-and-architecture/2.storage-design.md [hunk 1]
```diff
@@ -6,8 +6,7 @@
 
 ## 架构
 
-![](https://oscimg.oschina.net/oscnet/4b44fadbb1ee0455df6ed635d99154d2333.jpg)
-<!-- TODO: convert to github object -->
+![image](https://user-images.githubusercontent.com/42762957/71571213-2cd94c00-2b14-11ea-9b7f-a65067c61a63.png)
 图一  storage service 架构图
```
Comment (whitewum) [inline docs/manual-CN/1.overview/3.design-and-architecture/2.storage-design.md]: add line break

File: docs/manual-CN/1.overview/3.design-and-architecture/2.storage-design.md [hunk 2]
```diff
@@ -22,35 +21,35 @@
 
 对于点来说，**Nebula Graph** 使用不同的 Tag 表示不同类型的点，同一个 VertexID 可以关联多个 Tag，而每一个 Tag 都有自己对应的属性。对应到 kv 存储里面，**Nebula Graph** 使用 vertexID + TagID 来表示 key,  把相关的属性编码后放在 value 里面，具体 key 的 format 如图2 所示：
 
-![](https://oscimg.oschina.net/oscnet/524d35b78a3fa8178536c9710f39631ad38.jpg)
+![image](https://user-images.githubusercontent.com/42762957/71571002-1252a300-2b13-11ea-915f-b3fd16920b95.png)
 图二 Vertex Key Format
```
Comment (whitewum) [inline docs/manual-CN/1.overview/3.design-and-architecture/2.storage-design.md]: line break

File: docs/manual-CN/1.overview/3.design-and-architecture/2.storage-design.md [hunk 3]
```diff
@@ -22,35 +21,35 @@
...
 - `Tag ID` : 4 个字节, 用来表示关联的某个 tag
 - `Timestamp` : 8 个字节，对用户不可见，未来实现分布式事务 (MVCC) 时使用
 
 在一个图中，每一条逻辑意义上的边，在 Nebula Graph 中会建模成两个独立的 key-value，分别称为 out-key 和in-key。out-key 与这条边所对应的起点存储在同一个 partition 上，in-key 与这条边所对应的终点存储在同一个partition 上。通常来说，out-key 和 in-key 会分布在两个不同的 Partition 中。
 
 两个点之间可能存在多种类型的边，**Nebula Graph** 用 Edge Type 来表示边类型。而同一类型的边可能存在多条，比如，定义一个 edge type "转账"，用户 A 可能多次转账给 B， 所以 **Nebula Graph** 又增加了一个 Rank 字段来做区分，表示 A 到 B 之间多次转账记录。 Edge key 的 format 如图3 所示：
 
-![](https://oscimg.oschina.net/oscnet/3670bf726d129f52f72c1df7fd63f361aea.jpg)
+![image](https://user-images.githubusercontent.com/42762957/71571055-6a89a500-2b13-11ea-8953-e5c6d2588b16.png)
 图三 Edge Key Format
```
Comment (whitewum) [inline docs/manual-CN/1.overview/3.design-and-architecture/2.storage-design.md]: ditto

File: docs/manual-CN/1.overview/3.design-and-architecture/2.storage-design.md [hunk 4]
```diff
@@ -22,35 +21,35 @@
...
+- `Vertex ID` : 8 个字节，出边里面用来表示目标点的 ID， 入边里面表示源点的 ID。
 - `Timestamp` : 8 个字节，对用户不可见，未来实现分布式做事务的时候使用。
 
 针对 Edge Type 的值，若如果大于 0 表示出边，则对应的 edge key format 如图4 所示；若 Edge Type 的值小于 0，则对应的 edge key format 如图5 所示
 
-![](https://oscimg.oschina.net/oscnet/2a63e70a5c8bfdbde852c9be33037991e1d.jpg)
+![image](https://user-images.githubusercontent.com/42762957/71571153-e8e64700-2b13-11ea-9d96-ff7ac74db609.png)
 图4 出边的 Key Format
-![](https://oscimg.oschina.net/oscnet/e0975916b2689c148cab0cf938d903292f6.jpg)
+![image](https://user-images.githubusercontent.com/42762957/71571168-01eef800-2b14-11ea-8c86-c0696b966162.png)
```
Comment (whitewum) [inline docs/manual-CN/1.overview/3.design-and-architecture/2.storage-design.md]: ditto

**Comment by dutor** [comment]
Jenkins go

**Comment by dutor** [comment]
jenkins, go

**Pull request merged**